tests/functional/test_contract_event.py

```diff
@@ -78,14 +78,14 @@ def test_contract_logs_index_access(contract_instance, owner, assert_log_values)
     contract_instance.setNumber(2, sender=owner)
     contract_instance.setNumber(3, sender=owner)
 
-    assert_log_values(event_type[0], 1)
-    assert_log_values(event_type[1], 2)
-    assert_log_values(event_type[2], 3)
+    assert event_type[0] == contract_instance.NumberChange(None, 0, None, 1)
```

Comment:
I wasn't picturing support `None`, but I could see the benefit of it

@antazoey what do you think?

Reply:
wouldnt it be needed for full positional support?

Reply:
`None` makes sense as a skip for EVM languages... but if the language has a concept of `null` (not zero address or zero), we may desire a different behavior here.

🤔

Reply:
would `"*"` be better as the skip value?

```python
 assert event_type[0] == contract_instance.NumberChange("*", 0, "*", 1)
```

otherwise, using kwargs works too.
on the fence with `None`.

Reply:
you are right, good point.. and also `None` makes sense because it is fundamentally the same as a kwarg when not specified, so we have to do it that way